tests/unit/dataactcore/test_job_queue.py

```diff
@@ -0,0 +1,46 @@
+from dataactcore.utils.jobQueue import JobQueue
+from dataactcore.config import CONFIG_BROKER
+from unittest.mock import Mock
+from pytest import raises
+from dataactcore.utils.responseException import ResponseException
+import os.path
+
+
+def test_generate_d_file_success(monkeypatch):
+    """ Test successful generation of D1 and D2 files """
+    local_file_name = "12345_test_file.csv"
```

Comment:
The test here isn't cleaning up after itself, which means that we could delete lines 15 through 16 and have "passing" tests on subsequent runs. Some options:

* the  `[tmpdir](http://doc.pytest.org/en/latest/tmpdir.html)` fixture was made for this kind of work, but it'd be a bit tricky to integrate with our config system. Effectively, we'd want the tests to modify the `CONFIG_BROKER` at set up and reset it at tear down. This'd be super useful in the future, but might not be worth figuring out right now
* using a random file name helps us a little. The file's not deleted at the end of the test, but at least we'd generally catch errors